lib/xapi/problem.rb

```diff
@@ -48,6 +48,10 @@ def readme
       Readme.new(slug, path, setup).text
     end
 
+    def test_file
+      TestFile.new(path: language_dir, files: files).text
```

Comment:
Good call! Now that this has enough logic, it's nice to get it separated into it's own entity.